examples/05-custom-schema/05-latex-block/LaTex.tsx

```diff
@@ -0,0 +1,233 @@
+import {
+  useComponentsContext,
+} from "@blocknote/react";
+import { NodeView } from "prosemirror-view";
+import { BlockNoteEditor, propsToAttributes } from "@blocknote/core";
+import {
+  NodeViewProps,
+  NodeViewRenderer,
+  NodeViewWrapper,
+  ReactNodeViewRenderer,
+} from "@tiptap/react";
+import { createStronglyTypedTiptapNode, createInternalInlineContentSpec } from "@blocknote/core";
+import { mergeAttributes } from "@tiptap/core";
+import { ChangeEvent, useEffect, useState, useRef } from "react";
+import "./styles.css";
+
+function loadKaTex(callback: () => void) {
```

Comment:
I'd prefer to just have this as a dependency of the package, that makes the code a lot cleaner imo